lib/tree_sitter/node.rb

```diff
@@ -87,7 +89,7 @@ def respond_to_missing?(*args)
     # Iterate over a node's children.
     #
     # @yieldparam child [Node] the child
-    def each
+    def each(&_block)
```

Comment:
Why make it explicit?

Reply:
Sorbet is complaining about it, and passing it `--ignore` for `tree_sitter` files screws `TreeStand` typechecking.